docs/en/Running-Alluxio-on-GCE.md

```diff
@@ -96,26 +101,24 @@ $ export GCS_ACCESS_KEY_ID=<your access key>
 $ export GCS_SECRET_ACCESS_KEY=<your secret access key>
 ```
 
-Now you can launch the Alluxio cluster by running
-the script under `deploy/vagrant`:
+Now you can launch the Alluxio cluster by running the script under `deploy/vagrant`:
 
 ```bash
 $ ./create <number of machines> google
 ```
 
-
-Each node of the cluster runs an Alluxio worker, and the `AlluxioMaster` runs the Alluxio master.
+Each node of the cluster runs an Alluxio worker, and the `AlluxioMaster` node runs the Alluxio master.
 
 ## Access the cluster
 
-**Access through Web UI**
+**Access through the Web UI**
 
-After the command `./create <number of machines> google` succeeds, you can see two green lines like
+After the command `./create <number of machines> google` succeeds, you will see two green lines like
 below shown at the end of the shell output:
 
 ```bash
->>> AlluxioMaster public IP is xxx, visit xxx:19999 for Alluxio web UI<<<
->>> visit default port of the web UI of what you deployed <<<
+>>> AlluxioMaster public IP is xxx, visit xxxx:19999 for Alluxio web UI
```

Comment:
should we replace the first xxx with xxxx too?

Reply:
oops, made them both xxx.